src/pydap/model.py

```diff
@@ -287,18 +375,15 @@ class StructureType(DapType, Mapping):
     """A dict-like object holding other variables."""
 
     def __init__(self, name, attributes=None, **kwargs):
-        DapType.__init__(self, name, attributes, **kwargs)
+        super(StructureType, self).__init__(name, attributes, **kwargs)
 
-        # emulate a simple ordered dict
-        self._keys = []
-        self._dict = {}
+        # allow some keys to be hidden:
+        self._visible_keys = []
```

Comment:
OK, fair enough. I didn't realize that `SequenceType` can be directly fed lazy data in the form of structured arrays (records) rather than separate 1D arrays for columns.